scalapb-runtime-grpc/src/main/scala/scalapb/grpc/ConcreteProtoMethodDescriptorSupplier.scala

```diff
@@ -0,0 +1,19 @@
+package scalapb.grpc
+
+import com.google.protobuf.Descriptors
+import io.grpc.protobuf.{ProtoMethodDescriptorSupplier, ProtoServiceDescriptorSupplier}
+
+class ConcreteProtoMethodDescriptorSupplier(
+    fileDescriptor: Descriptors.FileDescriptor,
+    serviceName: String,
```

Comment:
instead of passing a string here and looking up by name below, let's pass the actual descriptors to the constructors (and return them directly through the getters below). See the various `javaDescriptorSource` methods in `DescriptorImplicits.scala`, please add such methods to `ServiceDescriptorPimp` and `MethodDescriptorPimp`